### ISSUE TITLE
Qubino Flush shutter ZMNHCDX calibration does not work correctly

### ISSUE THREAD

**Issue by scratchi** [Z-Wave-Me/ExpertUI#158]
Hello,

I have the following problem. I got a Qubino flush shutter (ZMNHCDX) that does not calibrate correctly. When looking for the device in "device description" the closest qubino device I can find is ZMNHCA2. However this description lacks some of the parameters in the configuration tab. For example it lacks the parameters 74 (motor moving up/down time) and 76 (motor operation detection). Parameter 74 however, could solve my problem with the incorrect calibration, as I could manually set the downtime here in order to get the correct downtime for my roller shutter. 

Therefore I'd like to ask if anyone has a suggestion how to find the parameter needed? Is there a better "device description" for my flush shutter that I could use? 

Thanks in advance for your replies!

Kind regards,
Fabian

**Comment by geirgp** [comment]
Hi Fabian,

I've got some Qubino devices myself, and the problem is that Qubino has not published their device descriptions to the "pepper1" database which Zway depends on. 

Until Qubino fixes this, which they told me in early December was already in motion, you will have to manually create the configuration file. Thats what I was told: https://forum.z-wave.me/viewtopic.php?f=3423&t=22451&p=59562&hilit=zddx#p59520

And I did it, waiting for zwave.me to include it in the next release (until then I have to manually add it to zway).

https://github.com/Z-Wave-Me/z-way-issues/issues/90
 
All the Qubino devices you currently see in zway are probably made manually by people who need them. Until Qubino publish their zddx files (device descriptors) to pepper1, you're unfortunately on your own..
